### DISCUSSION TITLE
Как при создании цикла передать туда параметр префильтра?

### DISCUSSION THREAD

**Discussion by gorbarov** [totumonline/totum-ru-issues-and-discussions#410] [Как это сделать?] [answered]
Привет, вопрос такой.

Есть таблица циклов с префильтром, который используется, чтобы открыть её отсортированную по этому префильтру через вызов linkToTable.

Задача, как при создании цикла записать значение префильтра в таблицу цикла?
В общем повторить по-сути стандартный механизм работы префильтров, когда таблица открывается через linkToTable.

**Comment by erpic-ru** [comment, marked as answer]
Мне кажется, что можно через кастомную кнопку добавления сделать и добавить переход в цикл сразу, например.

**Reply by gorbarov** [reply]
Спасибо, тоже об этом подумал, но захотелось сверится с практикующим сообществом )

Так а переход в цикл и так по умолчанию же происходит? Или я что-то упустил.

**Reply by erpic-ru** [reply]
Ну да, переход есть

**Reply by gorbarov** [reply]
@erpic-ru напомни плиз, где про кастомную кнопку почитать/посмотреть? Я помню, что видел где-то в видео и даже пересылал коллеге, но не могу найти )))

**Reply by gorbarov** [reply]
Нашёл, для будущих поколений.

Как создать кастомную кнопку добавления цикла.
https://www.youtube.com/watch?v=RX-EpctWUb0&list=PLfiZJiSJ6dosChEUL2nd6DX4JCLQ2oA83&index=2&ab_channel=totumonlineRU

**Reply by ttm-alexeypolunin** [reply]
Другой варианттв табл циклов делаете техническое поле, которое кодом при добавлении берет значения нужных префильтров. Оттуда кодом можно взять внутрь цикла. Важно помнить, что префильтры существуют только при действии из web-интерфейса.
